xarray/core/common.py

```diff
@@ -823,3 +823,25 @@ def ones_like(other, dtype=None):
     """Shorthand for full_like(other, 1, dtype)
     """
     return full_like(other, 1, dtype)
+
+
+def is_np_datetime_like(dtype):
+    """Check if a dtype is a subclass of the numpy datetime types
+    """
+    return (np.issubdtype(dtype, np.datetime64) or
+            np.issubdtype(dtype, np.timedelta64))
+
+
+def contains_netcdftime_datetimes(var):
+    """Check if a variable contains netcdftime datetime objects"""
+    try:
+        from netcdftime._netcdftime import datetime
+        return isinstance(var.data.flatten()[0], datetime)
```

Comment:
nit: move the `isinstance` line out of the try block (could go in an `else` block).